Sources/ArcGISToolkit/Components/OfflineMapAreasView/PreplannedMapModel.swift

```diff
@@ -98,17 +84,14 @@ public class PreplannedMapModel: ObservableObject, Identifiable {
         }
     }
     
-    /// Sets the model download preplanned offline map job if the job is in progress.
-    private func setDownloadJob() {
-        for case let preplannedJob as DownloadPreplannedOfflineMapJob in JobManager.shared.jobs {
```

Comment:
The for loop didn't break upon finding the job. This function also did much more than the name and doc.